spec/acceptance/foreman_basic_spec.rb

```diff
@@ -27,6 +27,9 @@
     class { '::apache::mod::passenger':
       manage_repo => false,
     }
+    class { 'redis::globals':
```

Comment:
This unfortunately doesn't work :
```
Warning: /Stage[main]/Redis::Config/File[/var/log/redis]: Skipping because of failed dependencies
       	Warning: /Stage[main]/Redis::Config/File[/var/opt/rh/rh-redis5/lib/redis]: Skipping because of failed dependencies
       	Warning: /Stage[main]/Redis::Ulimit/File[/etc/systemd/system/rh-redis5-redis.service.d/]: Skipping because of failed dependencies
       	Warning: /Stage[main]/Redis::Ulimit/File[/etc/systemd/system/rh-redis5-redis.service.d/limit.conf]: Skipping because of failed dependencies
       	Warning: /Stage[main]/Redis::Ulimit/Augeas[Systemd redis ulimit]: Skipping because of failed dependencies
       	Warning: /Stage[main]/Redis/Exec[systemd-reload-redis]: Skipping because of failed dependencies
       	Warning: /Stage[main]/Redis::Config/Redis::Instance[default]/File[/etc/opt/rh/rh-redis5/redis.conf.puppet]: Skipping because of failed dependencies
       	Warning: /Stage[main]/Redis::Config/Redis::Instance[default]/Exec[cp -p /etc/opt/rh/rh-redis5/redis.conf.puppet /etc/opt/rh/rh-redis5/redis.conf]: Skipping because of failed dependencies
       	Warning: /Stage[main]/Redis::Service/Service[rh-redis5-redis]: Skipping because of failed dependencies
```

Reply:
It seems to fail, because the package is already in :thinking:

Reply:
To be clear what I meant:
https://github.com/theforeman/puppet-foreman/blob/36aa5389b7da41d4770fa61a1c10a78239695735/spec/spec_helper_acceptance.rb#L26-L32
If you add `host.install_package('rh-redis5-redis')` there I think you'll avoid the SELinux idempotence issue.